scrapy/utils/test.py

```diff
@@ -30,7 +30,7 @@ def get_crawler(spidercls=None, settings_dict=None):
     from scrapy.spiders import Spider
 
     runner = CrawlerRunner(Settings(settings_dict))
```

Comment:
This is unrelated to these changes, but now that I see it we could pass `settings_dict` as it is to CrawlerRunner, we don't need to wrap it here into a Settings instance.